index.js

```diff
@@ -11,6 +11,18 @@ const changePrototype = (to, from) => {
 	Object.setPrototypeOf(to, fromPrototype);
 };
 
+const wrappedToString = (withName, fromBody) => `/* Wrapped ${withName}*/\n${fromBody}`;
+
+const toStringName = Object.getOwnPropertyDescriptor(Function.prototype.toString, 'name');
+const toStringDescriptors = Object.getOwnPropertyDescriptor(Function.prototype, 'toString');
+
+const changeToString = (to, from, name) => {
+	const withName = name === '' ? '' : `with ${name.trim()}() `;
+	const newToString = wrappedToString.bind(null, withName, from.toString());
+	Object.defineProperty(newToString, 'name', toStringName);
+	Object.defineProperty(to, 'toString', {...toStringDescriptors, value: newToString});
```

Comment:
We need to ensure `toString()` is not enumerable, otherwise it prints weird in Node.js.